pkg/controller/service/servicecontroller.go

```diff
@@ -118,22 +121,12 @@ func New(cloud cloudprovider.Interface, kubeClient clientset.Interface, clusterN
 		cache:            &serviceCache{serviceMap: make(map[string]*cachedService)},
 		eventBroadcaster: broadcaster,
 		eventRecorder:    recorder,
-		nodeLister: listers.StoreToNodeLister{
-			Store: cache.NewStore(cache.MetaNamespaceKeyFunc),
-		},
-		workingQueue: workqueue.NewDelayingQueue(),
+		nodeLister:       nodeInformer.Lister(),
+		nodeListerSynced: nodeInformer.Informer().HasSynced,
+		workingQueue:     workqueue.NewDelayingQueue(),
```

Comment:
This appears to be the only place that `NewDelayingQueue` or `NewRateLimitingQueue` is used (other than 1 test file).